src/navigation/mainTabs/MainTabNavigator.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import { useTheme } from '@shopify/restyle'
+import { Theme } from '../../theme/theme'
+import Hotspots from '../../features/hotspots/HotspotsNavigator'
+import { TabBarIconType, MainTabType } from './tabTypes'
+import TabBarIcon from './TabBarIcon'
+
+const MainTab = createBottomTabNavigator()
+
+const Main = () => {
+  const {
+    colors: { secondaryBackground },
+  } = useTheme<Theme>()
+
+  return (
+    <MainTab.Navigator
+      tabBarOptions={{
+        showLabel: false,
+        style: {
+          backgroundColor: secondaryBackground,
+          borderTopWidth: 0,
+        },
+      }}
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused, color, size }: TabBarIconType) => {
+          return (
+            <TabBarIcon
+              name={route.name as MainTabType}
+              focused={focused}
+              color={color}
+              size={size}
+            />
+          )
+        },
+      })}
+    >
+      <MainTab.Screen name="Account" component={Hotspots} />
+      <MainTab.Screen name="Hotspots" component={Hotspots} />
+      <MainTab.Screen name="Network" component={Hotspots} />
```

Comment:
I know this is just a stub, but we probably want to steer in the direction of the new designs here. The four tabs in the new app are currently, Hotspots, Account, Notifications and More

![image](https://user-images.githubusercontent.com/792845/100647871-f93f0d00-32f4-11eb-8986-1b356d3fe589.png)

Reply:
I guess "Wallet" is what we'll probably call the second tab based on: https://github.com/helium/hotspot-app/issues/15